udfs/community/nlp_compromise_number.sql

```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
  */
 
-CREATE OR REPLACE FUNCTION fn.nlp_compromise_number(str STRING)
+CREATE OR REPLACE FUNCION fn.nlp_compromise_number(str STRING)
```

Comment:
Is this typo of FUNCTION?

Reply:
it was a temporary intentional typo introduced to test the build process. reverted now